enos/modules/vault_verify_unsealed/templates/verify-vault-node-unsealed.sh

```diff
@@ -12,9 +12,11 @@ fail() {
 test -x "$binpath" || fail "unable to locate vault binary at $binpath"
 
 export VAULT_ADDR='http://127.0.0.1:8200'
-export VAULT_TOKEN='${vault_token}'
 
+health_status=$(curl http://127.0.0.1:8200/v1/sys/health |jq '.')
```

Comment:
Interesting! I guess the status endpoint doesn't need a token. Makes sense as you can run it before init.